example3/Microsoft.Diagnostics.Metric/AggregationConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Metric
+{
+    public static class AggregationConfigurations
```

Comment:
Would .NET want to expose this? or just expose listener , and its upto listeners (Otel etc) to build aggregators..

Reply:
I think its at least on the table that .NET exposes stuff like this. In this case these types aren't implementations of aggregators, only descriptions of what aggregation the instrumenting dev recommends to occur. I envision use-cases where a library developer wants to communicate things like "I recommend  tracking these times as percentiles reporting the 50th, 95th, and 99th percentile and bucket requests by response code". A listener isn't obligated to do what the instrumenter asks, but regardless the instrumenter still needs some API to describe their recommendation.

Reply:
thanks for clarifying. yes this makes sense.

Reply:
This should be part of the "hinting" rather than specific to Aggregation.  @noahfalk, I though this was more for the .NET specific aggregator in the "SDK" portion (.NET runtime consumption of Listener rather than OTel) of .NET API?

Reply:
Another potentially important hint will be Units. Maybe Monotonic is a hint as well. Maybe Min/Max limits/thresholds, etc...
 
Bucket configuration probably fits with Histogram aggregation, maybe.

Maybe we have multiple aggregators at the same time.  I.e. We want Sum, Histogram, and Summary together.